lib/app.js

```diff
@@ -70,6 +67,14 @@ function isDesktop() {
   //return window.matchMedia( "(min-width: 800px)" ).matches;
 }
 
+function urlParams() {
+  return document.location.search.slice(1).split('&').reduce(function(acc, x) {
+    var parts = x.split('=');
+    acc[decodeURIComponent(parts[0])] = decodeURIComponent(parts[1]);
+    return acc;
+  }, {});
+}
+
```

Comment:
nit: semantic function name would be nice, esp since no doc comments. eg getURLParamsAsHash or something like that. #EIBTI